tarodb/asset_minting.go

```diff
@@ -277,6 +283,28 @@ func (a *AssetMintingStore) CommitMintingBatch(ctx context.Context,
 
 		return nil
 	})
+
+	// Handle sqlite asset seedling name unique constraint violation error.
+	var sqliteErr *sqlite.Error
```

Comment:
As discussed offline: Would be good to abstract into more general error checking that handles both DB backends.